code/game/machinery/iv_drip.dm

```diff
@@ -9,6 +9,20 @@
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 	var/obj/item/reagent_containers/iv_bag/bag = null
 
+/obj/machinery/iv_drip/Initialize(mapload)
+	. = ..()
+	if(mapload)
+		if(bag) // just in case anyone in the future ever adds IV drip subtypes with bags attached
+			return
+		for(var/obj/item/I in loc)
+			if(istype(I, /obj/item/reagent_containers/iv_bag))
+				I.forceMove(src)
+				bag = I
+				update_icon()
+				START_PROCESSING(SSmachines, src)
+				break
```

Comment:
The `update_icon()` call is superfluous as its called on 24.

I also feel like this should be its own proc for readability but ill let the other reviewers decide.